### PR TITLE
Updated vuetify to 2.1.0

### PR DESCRIPTION
**Pull request by MikeLenoir**

### CODE REVIEW

**Comment by kinow** [comment]
Hi @MikeLenoir I think you didn't execute the `npm install` that would change the lock file. I think there's another PR for the same issue: #277, created some moments ago.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -50,39 +50,39 @@ control logs or otherwise.)
...
+prepare derivative works of, publicly display, publicly perform,
+sub-licence, and distribute this contribution and such modifications
+and derivative works consistent with this project or the licence(s)
+involved or other appropriate open source licence(s) specified by
+the project and approved by the
+[Open Source Initiative (OSI)](http://www.opensource.org/).
 
 (f) If I become aware of anything that would make any of the above
-    inaccurate, in any way, I will let NIWA know as soon as
-    I become aware.
```
Comment (kinow) [inline CONTRIBUTING.md]: These changes to `CONTRIBUTING.md` are probably not required for this update.

Comment (MikeLenoir) [reply CONTRIBUTING.md]: Sorry confused, should I undo the last commits of adding my name?

Comment (kinow) [reply CONTRIBUTING.md]: Not that part. You added your name, but probably your editor formatted the whole file, changing some lines (GitHub UI highlights changes that add content as green, and changes that remove content as red, as the lines above my comment)

**Pull request closed without merging**